src/util/generate/generate-dir-map.ts

```diff
@@ -0,0 +1,64 @@
+import fs from 'fs'
+import { promisify } from 'util'
+import { join, parse } from 'path'
+import { allDetectors } from './detect-from-manifests'
+import ignore from '../ignored'
+
+const ignored = ignore.split('\n')
+const readdir = promisify(fs.readdir)
+const stat = promisify(fs.stat)
+
+export type dirMap = {
```

Comment:
```suggestion
export type DirMap = {
```

Types should start with a capital letter